spec/scrabble_spec.rb

```diff
@@ -0,0 +1,88 @@
+require "spec_helper"
+require "./lib/scrabble"
+
+describe Scrabble::Scrabble do
+  # expected functionality of Scrabble class
+
+  describe "#is_valid?" do
```

Comment:
This is a good set of tests - the only thing is that you may not necessarily want to expose the `is_valid?` method outside of your scoring methods. Since the score methods are supposed to be using the logic in `is_valid?` it would also be a useful test to check that the error message comes from the score methods as well.